packages/web/server/Announcement.ts

```diff
@@ -0,0 +1,37 @@
+import getConfig from 'next/config'
+import airtableInit from '../server/airtable'
```

Comment:
I think you should be able to do absolute paths here with the .babelrc - no?

Reply:
so far i havent been able to do absolute paths for anything in the server folder.

Reply:
yeah the linter in VScode is cool with it but the app crashes